src/components/dialogs/parameters/common/parameters-creation-dialog.tsx

```diff
@@ -97,13 +95,13 @@ const CreateParameterDialog: React.FunctionComponent<CreateParameterProps> = ({
     const onSubmit = useCallback(
         (values: FormData) => {
             createParameter(
-                formatNewParams(parameterGetValues()),
+                parameterFormatter(parameterValues()),
```

Comment:
it would be add a .catch to handle errors or put a log in the console

Reply:
I only changed the name to be more generic.